src/App.tsx

```diff
@@ -11,12 +11,26 @@ import logo from "./logo.svg";
 // This will load the translated strings in FBT.
 init({ translations: intl });
 
-IntlViewerContext.locale = "fr_FR";
+
 
 interface LanguageSelectorProps {
   name: string;
 }
 
+const supportedLanguages: { [shortname: string]: { name: string } } = {
+  "en_US": { name: "English" },
+  "fr_FR": { name: "French" },
+  "pi_PI": { name: "Pirate" }
+}
+
+const langStore = window.localStorage;
+if (langStore.getItem('lang') == null) {
```

Comment:
Let's use " everywhere instead of '

Reply:
okay! I wasn't too sure what the difference between the two were, because I saw it being used in sample stuff for full strings, and in java you only use ' ' for characters. But I'll use " from now on